docs/source/api.rst

```diff
@@ -25,21 +25,12 @@ There's also a HAWC client available:
 
 .. code-block:: python
 
-    from hawc.client import Client
+    from hawc_client import HawcClient
 
-    client = Client(
-        email="me@me.com",
-        password="keep-it-hidden",
-        root_url="https://hawcproject.org"
-    )
+    client = HawcClient(root_url="https://hawcproject.org")
+    client.authenticate(username="username",password="password")
 
-    hero_ids = list(range(5000, 5020))
-    client.lit_import_hero(
-        assessment_id=123,
-        title="example title",
-        description="example description",
-        ids=hero_ids
-    )
+    client.animal.endpoints(assessment_id=123)
```

Comment:
let's use `lit.references()`  instead, that will always be a snappy response.

maybe demo both a read and a write:

```python
# as an example, get all references for an assessment
client.lit.references(assessment_id=123)

# as an example, import new references to an assessment
client.lit_import_hero(
    assessment_id=123,
    title="example title",
    description="example description",
    ids=[5000, 5010]
)
```